bodhi/server/templates/fragments.html

```diff
@@ -79,14 +79,12 @@
 % if comment.user.name == "bodhi":
```

Comment:
Question, what is the reason to have a different rendering for comments by 'bodhi'? Reading the code, it seems this would work fine for ordinary users.

Reply:
the "bodhi" user is the one that adds the status update comments for an update.